$OpenBSD$

I don't know how to do meson. This does the right thing.

Index: meson_options.txt
--- meson_options.txt.orig
+++ meson_options.txt
@@ -3,6 +3,6 @@ option(
   'plugins',
   type: 'array',
   choices: [ 'lastfm', 'audioplayer', 'cdrom', 'ipod' ],
-  value: [ 'lastfm', 'audioplayer', 'cdrom', 'ipod' ]
+  value: [ 'audioplayer', 'cdrom', 'ipod' ]
 )
 option ('documentation', type : 'boolean', value : false)
